lib/sms.rb

```diff
@@ -0,0 +1,31 @@
+require "twilio-ruby"
+
+class SMS
+
+  TIME_FORMAT = "%H:%M"
+
+  def initialize(config, client: nil)
+    @client = client || Twilio::REST::Client.new(config[:acount_sid, config[:auth_token]])
```

Comment:
Metrics/LineLength: Line is too long. [90/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)